src/test/java/com/wikia/webdriver/testcases/adstests/TestUap.java

```diff
@@ -86,6 +88,8 @@ private void verifySlotsBlocked(AdsBaseObject ads, List<Map<String, Object>> slo
   private void verifySlotsUnblocked(AdsBaseObject ads, List<Map<String, Object>> slotsData) {
     for (Map<String, Object> slotData : slotsData) {
       String slotName = slotData.get("slotName").toString();
+      ads.wait.forElementPresent(By.cssSelector("#"+slotName));
+      ads.scrollToPosition("#"+slotName);
```

Comment:
Same as above, try to use By. class and add different implementation of scrollToPosition if it will be needed